serpentTools/objects/containers.py

```diff
@@ -39,54 +50,84 @@ def __init__(self, container, name, bu, step, day):
         self.b1Unc = {}
         self.infUnc = {}
 
-    def set(self, variablename, variablevalue, uncertainty=False):
+    def set(self, variableName, variableValue, uncertainty=False):
         """
+        Sets the value of the variable and, optionally, the associate s.d.
+
         Parameters
         ----------
-        variablename:   Variable Name
-        variablevalue:  Variable Expected Value
-        uncertainty:    Boolean Variable- set to True in order to retrieve the
-                        uncertainty associated to the expected values
+        variableName: str
+            Variable Name
+        variableValue: str/tuple/list
+            Variable Value
+        uncertainty:   bool
+            Boolean Variable- set to True in order to retrieve the
+            uncertainty associated to the expected values
+
+        Notes:
+        ---------
+            Raises a warning if the value of the variable is overwritten
+
         """
 
         # 1. Check the input type
-        variablename = _SupportingObject._convertVariableName(variablename)
+        variableName = _SupportingObject._convertVariableName(variableName)
         if not isinstance(uncertainty, bool):
-            raise messages.error("Uncertainty must be a boolean variable")
+            raise messages.error('The variable uncertainty has type %s.\n ...'
+                                 'It should be boolean.', type(uncertainty))
         # 2. Pointer to the proper dictionary
-        setter = self._lookup(variablename, uncertainty)
+        setter = self._lookup(variableName, uncertainty)
         # 3. Check if variable is already present. Then set the variable.
-        if variablename in setter:
+        if variableName in setter:
             messages.warning('The variable will be overwritten')
-        setter[variablename] = variablevalue
+        setter[variableName] = variableValue
 
-    def get(self, variablename, uncertainty=False):
+    def get(self, variableName, uncertainty=False):
+        """
+        Gets the value of the variable VariableName from the dictionaries
 
+        Parameters
+        ----------
+        variableName: str
+            Variable Name
+        uncertainty:   bool
+            Boolean Variable- set to True in order to retrieve the
+            uncertainty associated to the expected values
+
+        Returns
+        -------
+        x: str/tuple/list
+            Variable Value
+        dx: str/tuple/list
+            Associated uncertainty
+
+        """
         # 1. Check the input values
-        variablename = _SupportingObject._convertVariableName(variablename)
+        variableName = _SupportingObject._convertVariableName(variableName)
```

Comment:
Please remove.
This is only necessary if the variable name requested is SERPENT_STYLE. We will make it known to the user that the variables are __stored__ with mixedCase names.